release_docs/INSTALL_Auto.txt

```diff
@@ -401,8 +350,10 @@ CONTENTS
             --enable-hl, --enable-cxx, --enable-fortran, --enable-java
 
 
-4.3.11. Backward compatibility
-        The 1.14 version of the HDF5 library can be configured to operate
+   3.11. Backward compatibility
+        The 1.15 version of the HDF5 library can be configured to operate
```

Comment:
Should this sentence be in the 1.14 branch?

Reply:
Of course not it should be 1.14 and the next two lines for 1.14 api should be removed

Reply:
There are 3 more of them in HDF5Examples/CMakeUserPresets.json lines 22 - 27:

      "environment": {
        "HDF5_ROOT": "${sourceParentDir}/temp/HDF_Group/HDF5/1.15.0",
        "HDF5_PLUGIN_PATH": "${sourceParentDir}/temp/HDF_Group/HDF5/1.15.0/lib/plugin"
      },
      "cacheVariables": {
        "HDF5_ROOT": {"type": "STRING", "value": "${sourceParentDir}/temp/HDF_Group/HDF5/1.15.0"},

They were already there in the hdf5_1_14 branch.  I'll change them to 1.14.4, but @byrnHDF  do we need to add the file/lines to h5vers?